packages/opc-nav/declaration.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '*.scss';
```

Comment:
Types seems to be incomplete

Reply:
Ohhh i missed this. I think this needs to be automated rather than a manual process.

Reply:
Done